gef-ui/server/utils/facility-helpers.test.js

```diff
@@ -363,3 +365,23 @@ describe('summaryIssuedUnchanged()', () => {
     expect(result).toEqual(false);
   });
 });
+
+describe('issuedFacilityConfirmation()', () => {
+  it('Should return `false` as the facility has not been issued', () => {
+    expect(issuedFacilityConfirmation(MOCK_AIN_APPLICATION_UNISSUED_ONLY)).toEqual(false);
+  });
+  it('Should return `false` as the submission type is AIN', () => {
+    expect(issuedFacilityConfirmation(MOCK_AIN_APPLICATION_ISSUED_ONLY)).toEqual(false);
+  });
+  it('Should return `false` as the submission count is one', () => {
+    expect(issuedFacilityConfirmation(MOCK_MIA_APPLICATION_UNISSUED_ONLY)).toEqual(false);
+  });
+  it('Should return `true` as the facility has been issued, application is MIN and submission count is greater then two', () => {
+    const mockMIN = {
+      ...MOCK_AIN_APPLICATION_ISSUED_ONLY,
+      submissionType: CONSTANTS.DEAL_SUBMISSION_TYPE.MIN,
+      submissionCount: 3,
+    };
+    expect(issuedFacilityConfirmation(mockMIN)).toEqual(false);
```

Comment:
test says it should return true, but `toEqual` has false value

Reply:
Updated test with an extra test case.